pom.xml

```diff
@@ -106,6 +106,11 @@
 			<artifactId>thymeleaf-layout-dialect</artifactId>
 		</dependency>
 
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-actuator</artifactId>
```

Comment:
Note for infra-team: https://www.baeldung.com/spring-boot-actuators , it provides probes for Docker healthcheck/Kubernetes probes.
Thanks @alecharp !